config/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	APIGatewayV2                    ResourceType               `yaml:"APIGatewayV2"`
 	AccessAnalyzer                  ResourceType               `yaml:"AccessAnalyzer"`
 	AutoScalingGroup                ResourceType               `yaml:"AutoScalingGroup"`
+	AppRunnerService                ResourceType               `yaml:"AppRunnerService"`
```

Comment:
You probably need to update the `config_test.go` file to pass the unit test.

Reply:
actually you right - but seem like that test is also failing on master for me :| will fix it today

Reply:
sorted, but I needed to add `field names` to the test file, this way it should be easier to maintain when new fields will be added.